bert_e/workflow/gitwaterflow/integration.py

```diff
@@ -136,6 +136,24 @@ def update(wbranch, source):
         prev = branch
 
 
+def check_integration_branches(job):
+    """Check if the integration branches can be created."""
+
+    approvals = set(job.pull_request.get_approvals())
+    approved_by_author |= job.pull_request.author in approvals
+    if (approved_by_author is True and
+        len(job.git.cascade.dst_branches) > 1):
+        create_integration_branches()
```

Comment:
integration branches will be created later on, this function shouldn't be called here.